CHANGELOG.rst

```diff
@@ -0,0 +1,22 @@
+===============================
+Ansible Network: network-engine
+===============================
+
+2.5.0
+=====
+
+Major Changes
+-------------
+
+- Initial release of the ``network-engine`` Ansible role.
+
+- This role provides the foundation for building network roles by providing modules and plugins that are common to all Ansible Network roles. All of the artifacts in this role can be used independent of the platform that is being managed.
```

Comment:
2nd sentence: "This role can be used on any network platform or on multiple network platforms." (if that is, in fact, true)

Reply:
Let's discuss this